src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final Bulldozer caterpillar = new Bulldozer();
+    private final Bulldozer komatsu = new Bulldozer();
+    private final Bulldozer johnDeere = new Bulldozer();
+
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(caterpillar, komatsu, johnDeere);
```

Comment:
Producer means: it produces new instances of machines every time method 'get' is invoked. 
It's better not to return same bulldozers each time, but to create new instances.

Change it also for Excavator and Truck